src/components/LoadingSkeletons/BarChartLoadingSkeleton.tsx

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import LoadingMessage from '../LoadingMessage/LoadingMessage';
+import { cxBackgroundGray } from './LoadingSkeletonsSvgUtil';
+import LoadingSkeleton, {
+	ILoadingSkeletonProps,
+	IStandardSkeleton,
+} from './LoadingSkeleton';
+
+export const BarChartSkeleton = (
+	props: IStandardSkeleton
+): React.ReactElement => {
+	const { width, height, className } = { ...props };
```

Comment:
Do we need to create a new object here by spreading?
`const { width, height, className } = props ;`

Reply:
Also, can we provide some default values/ mandatory field requirements ?

Reply:
> Do we need to create a new object here by spreading?
> `const { width, height, className } = props ;`

We do not. But it is the way it's done everywhere else in Lucid and it's done for consistency. Please see comment from Noah Yasskin above

Reply:
We do not. But it is the way it's done everywhere else in Lucid and it's done for consistency. Please see comment from Noah Yasskin above

Reply:
defaults - Done